addons/product/product.py

```diff
@@ -539,6 +539,7 @@ def _get_product_template_type(self, cr, uid, context=None):
             help="This is minimum quantity to purchase from Main Vendor."),
         'seller_id': fields.related('seller_ids','name', type='many2one', relation='res.partner', string='Main Vendor',
             help="Main vendor who has highest priority in vendor list."),
+        'seller_price': fields.related('seller_ids','price', type='float', string='Vendor Price', help="Purchase price from from Main Vendor."),
 
```

Comment:
must have a computed field for supplier info: default = first, if vendor or qty min given in the context will take the first one that match. 
existing seller_* fields are related on that computed field